randovania/games/prime1/json_data/header.txt

```diff
@@ -18,29 +38,20 @@ Dock Weaknesses
       All of the following:
           Ice Beam or Plasma Beam or Power Beam or Wave Beam
           Any of the following:
-              Combat Visor or Thermal Visor or X-Ray Visor
+              Can Use Arm Cannon
               Morph Ball Bomb and Morph Ball and Scan Visor
```

Comment:
is there a reason this template isn't just `Shoot Any Beam OR (Morph Ball AND Morph Ball Bomb AND Scan Visor)?